src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -1,79 +1,1419 @@
 // this is the IRS Plan page component
-import reducerRegistry from '@onaio/redux-reducer-registry';
+import { Actions } from 'gisida';
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { RouteComponentProps } from 'react-router';
+import { Column } from 'react-table';
+import {
+  Button,
+  Col,
+  Form,
+  FormGroup,
+  Input,
+  InputGroup,
+  InputGroupAddon,
+  Label,
+  Row,
+} from 'reactstrap';
 import { Store } from 'redux';
 
+import GeojsonExtent from '@mapbox/geojson-extent';
+import DrillDownTable, { DrillDownProps, DropDownCell } from '@onaio/drill-down-table';
+import reducerRegistry from '@onaio/redux-reducer-registry';
 import superset from '@onaio/superset-connector';
 
-import { SUPERSET_PLANS_TABLE_SLICE } from '../../../../../configs/env';
-import { HOME, HOME_URL, INTERVENTION_IRS_URL } from '../../../../../constants';
-import { RouteParams } from '../../../../../helpers/utils';
+import { GREY } from '../../.../../../../../colors';
+import {
+  SUPERSET_JURISDICTIONS_DATA_SLICE,
+  SUPERSET_JURISDICTIONS_SLICE,
+  SUPERSET_PLAN_STRUCTURE_PIVOT_SLICE,
+  SUPERSET_PLANS_TABLE_SLICE,
```

Comment:
I think `GREY`, `extractPlanRecordFromPlanPayload`, `PlanRecordResponse`, and `SUPERSET_PLANS_TABLE_SLICE` are imported but not used.

Reply:
Ooh yeah, I'll make sure to remove any un-used imports